datasets/modeled/buildings/dimension_mappings/emm_to_county.csv

```diff
@@ -0,0 +1,3109 @@
+from_id,to_id,from_fraction
```

Comment:
@asatremeloy I added the from_fraction value with this Spark code:
```
    df.groupBy("from_id").agg(F.count("from_id").alias("count")).withColumn(
        "from_fraction", 1 / F.col("count")
    ).drop("county").join(df, on="from_id").select(
        "from_id", "to_id", "from_fraction"
    ).sort(
        "from_id", "to_id"
    ).toPandas().to_csv(
        "emm_to_county_with_fraction.csv", index=False
    )
```

Reply:
@daniel-thom Great, thanks for sharing that. Looks good to me.